vpr/src/draw/intra_logic_block.cpp

```diff
@@ -34,11 +34,38 @@
 #include "draw.h"
 #include "draw_triangle.h"
 
+constexpr float FRACTION_TEXT_PADDING = 0.01;
```

Comment:
Should put a comment above this briefly explaining what it is for and what it is a fraction of.